tests/platform_tests/test_cont_warm_reboot.py

```diff
@@ -126,6 +144,21 @@ def test_cont_warm_reboot(duthost, localhost, conn_graph_facts, continuous_reboo
         if "disabled" in issu_capability:
             pytest.skip("ISSU is not supported on this DUT, skip this test case")
 
-    for _ in range(continuous_reboot_count):
+    # Start advancedReboot script on the ptf host to enable continuous I/O
+    advancedReboot = get_advanced_reboot(rebootType='warm-reboot', enableContinuousIO=enable_continuous_io)
```

Comment:
I believe with this change the AdvancedReboot class/Fixture name should be changed as it is not doing reboot anymore!

Reply:
Based on the discussion offline - In general, the AdvancedReboot class(and in turn advanced-reboot script) is doing more than just reboot - starting I/O, monitor control/data plane, verifies BGP sessions and more.
The script now needs to be broken down to meaningful modules for simplicity and better organization.
These changes are outside the context of the PR1880 and will be taken care of in a different PR in the near future.